pkg/server/server.go

```diff
@@ -594,26 +661,9 @@ func (s *ProxyServer) serveRecvFrontend(frontend *GrpcFrontend, recvCh <-chan *c
 		}
 	}
 
-	// TODO: Log an error if the frontend stream is closed with an established tunnel still open.
-	klog.V(5).InfoS("Close streaming", "connectionID", firstConnID)
-
+	klog.V(5).InfoS("Close frontend streaming", "connectionID", firstConnID, "streamUID", frontend.streamUID)
 	if backend == nil {
 		klog.V(2).InfoS("Streaming closed before backend intialized")
```

Comment:
should this be moved to the defer at the top of this method? There are a bunch of places we exit the loop with a return, which would skip these log lines.

Reply:
Good idea, done.